acceptance/tests/resource/service/launchd_provider.rb

```diff
@@ -79,4 +79,19 @@ def launchctl_assert_status(host, service, expect_running)
   ensure_service_on_host(agent, svc, {:enable => 'true'}) do
     launchctl_assert_status(agent, svc, false)
   end
+
+  # switching from stopped to running should output the correct status of the service and not 'absent'
+  step "Start the service on #{agent} when service is stopped, and check outoput"
+  on agent, puppet_resource('service', 'com.puppetlabs.sloth', 'ensure=stopped')
```

Comment:
This is equivalent to `on agent, puppet_resource('service', svc, 'ensure=stopped')`. You can rely on `svc` variable that is already defined